Makefile

```diff
@@ -27,85 +30,26 @@ pip-check:
 # Build and run production docker container
 
 build-prod:
-	docker build --file=Dockerfile . --tag=openslides-backend
+	docker build . --tag=openslides-backend
 
-run-prod:
+run-prod: | build-prod
 	docker run --interactive --tty \
 	--publish 9002:9002 --publish 9003:9003 --rm openslides-backend
 
 
-# Build and run development and testing docker container
-
-build-dev:
-	docker build --file=Dockerfile-dev . --tag openslides-backend-dev
-
-mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
-
-action_args=--interactive \
-	--volume=$(dir $(mkfile_path))openslides_backend:/srv/code/openslides_backend \
-	--volume=$(dir $(mkfile_path))tests:/srv/code/tests \
-	--network=host --rm openslides-backend-dev
-dev_args=--tty $(action_args)
-
-run-dev: | build-dev
-	docker run $(dev_args)
-
-run-dev-interactive run-bash: | build-dev
-	docker run $(dev_args) sh
-
-run-cleanup: | build-dev
```

Comment:
Can you add it back? We must have a make target to run the cleanup scripts inside the containers without starting everything (e.g. for ludwig - he does not have python installed).

Reply:
Inside the container (started with docker compose) you can just type `make` to get the cleanup. `make run-cleanup` is not required any more.

Reply:
Ok. This is just a comfy-thing for me personally. Sometimes I do not want to start everything to run cleanup ;)

Reply:
;-)

Ok I will provide you a private script in the dev directory ...